src/main/java/org/tikv/common/ConfigUtils.java

```diff
@@ -67,6 +67,7 @@
   public static final String TIKV_RAWKV_SCAN_TIMEOUT_IN_MS = "tikv.rawkv.scan_timeout_in_ms";
   public static final String TIKV_RAWKV_CLEAN_TIMEOUT_IN_MS = "tikv.rawkv.clean_timeout_in_ms";
   public static final String TIKV_BO_REGION_MISS_BASE_IN_MS = "tikv.bo_region_miss_base_in_ms";
+  public static final String TIKV_SLOW_LOG_THRESHOLD = "tikv.slow_log_threshold";
```

Comment:
how to configure it to an individual log file?

Reply:
by log4j config file

Reply:
https://www.jianshu.com/p/ccafda45bcea

Reply:
got it